src/components/entity/state-info.html

```diff
@@ -61,22 +61,33 @@
   is: 'state-info',
 
   properties: {
-    detailed: {
-      type: Boolean,
-      value: false,
-    },
-
     stateObj: {
       type: Object,
     },
 
     inDialog: {
       type: Boolean,
+      value: false,
     },
   },
 
-  computeStateName: function (stateObj) {
-    return window.hassUtil.computeStateName(stateObj);
+  _getContext: function () {
```

Comment:
Instead of this search function, I think we should pass down the groupEntity or even just the context string from ha-entities-card. It touches more files, but this is currently bypassing Polymer's dependency tracking.